src/deluge/gui/views/automation_instrument_clip_view.cpp

```diff
@@ -2315,9 +2412,10 @@ void AutomationInstrumentClipView::modEncoderAction(int32_t whichModEncoder, int
 
 					modelStack->getTimelineCounter()->instrumentBeenEdited();
 
```

Comment:
updated rendering code for updating value on display when adjusting parameter values with mod encoders